src/scss/modifiers.scss

```diff
@@ -0,0 +1,67 @@
+// BEM style modifiers for the most generic cases, e.g. single property of HTML element
+// Useful for programmatic switching on/off as well as rapid development prototyping
+
+// Element classes should be the name of the element preceded with a dot and all modifiers for it put inside
+
+@import "scss/constants.scss";
+
+// set some generic maps and enums here for use in props
+$text-align-enum: (left, center, right);
+$spacing-map: (
+  xs: $spacing--xs,
+  sm: $spacing--sm,
+  md: $spacing--md,
+  lg: $spacing--lg,
+  xl: $spacing--xl,
+  xxl: $spacing--xxl,
+);
+
+// generic placeholder for any element
+// helps to not duplicate .div--text-left, .input--text-left...
+.mod {
+  &--not-implemented {
+    display: none;
+    visibility: hidden;
+  }
+
+  &--text {
+    @each $where in $text-align-enum {
+      &-#{$where} {
+        text-align: $where;
+      }
+    }
+  }
+
+  @for $step from 1 through 20 {
+    &--width-#{5 * $step}pp {
+      width: 5 * $step * 1%;
+    }
+
+    &--width-#{2 * $step}em {
+      width: 2 * $step * 1em;
+    }
+
+    &--height-#{5 * $step}pp {
+      height: 5 * $step * 1%;
+    }
+
+    &--height-#{$step}em {
+      height: $step * 1em;
+    }
+
+    &--font-weight-#{50 * $step} {
+      font-weight: 50 * $step;
+    }
+
+    &--line-height-#{10 * $step + 5} {
+      line-height: 0.1 * $step + 0.5;
+    }
+  }
+}
+
+// specific tags modifiers
+.div {
+}
+
+.input {
```

Comment:
these are empty, so do we still need them?

Reply:
@alexey-petrov Do note you can also insert a suggestion for them to be deleted, like this https://github.com/sparkletown/sparkle/pull/1705/files/a3835da5ae0f7d54afb9effca82fd8391cc61837#r685717511

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1705/commits/3ca884136419f135a2893c7e0fc2ee4af1d3301b